src/main/scala/io/github/spark_redshift_community/spark/redshift/Parameters.scala

```diff
@@ -292,5 +293,10 @@ private[redshift] object Parameters {
      * include in the COPY command (e.g. `COPY "PUBLIC"."tablename" ("column1" [,"column2", ...])`)
      */
     def includeColumnList: Boolean = parameters("include_column_list").toBoolean
+
+    /**
+     * The AWS SSE-KMS key to use for encryption during UNLOAD operations instead of AWS's default encryption
```

Comment:
travis build failed with `/home/travis/build/spark-redshift-community/spark-redshift/src/main/scala/io/github/spark_redshift_community/spark/redshift/Parameters.scala:298: File line length exceeds 100 characters`

Reply:
This is fixed now.